src/legacy/core_plugins/console/public/src/components/settings_modal.tsx

```diff
@@ -0,0 +1,272 @@
+              }}
+            />
+          </EuiFormRow>
+          <EuiFormRow>
+            <EuiSwitch
+              checked={wrapMode}
+              data-test-subj="settingsWrapLines"
+              id="wrapLines"
+              label={
+                <FormattedMessage
+                  defaultMessage="Wrap long lines"
+                  id="console.settingsPage.wrapLongLinesLabelText"
+                />
+              }
+              onChange={e => setWrapMode(e.target.checked)}
+            />
+          </EuiFormRow>
+
+          <EuiFormRow
+            label={
+              <FormattedMessage
+                id="console.settingsPage.jsonSyntaxLabel"
+                defaultMessage="JSON syntax"
+              />
+            }
+          >
+            <EuiSwitch
+              checked={tripleQuotes}
+              data-test-subj="tripleQuotes"
+              id="tripleQuotes"
+              label={
+                <FormattedMessage
+                  defaultMessage="Use triple quotes in output pane"
+                  id="console.settingsPage.tripleQuotesMessage"
+                />
+              }
+              onChange={e => setTripleQuotes(e.target.checked)}
+            />
+          </EuiFormRow>
+          <EuiFormRow
+            labelType="legend"
+            label={
+              <FormattedMessage
+                id="console.settingsPage.autocompleteLabel"
+                defaultMessage="Autocomplete"
+              />
+            }
+          >
+            <EuiCheckboxGroup
+              options={autoCompleteCheckboxes}
+              idToSelectedMap={checkboxIdToSelectedMap}
+              onChange={(e: any) => {
+                onAutocompleteChange(e as AutocompleteOptions);
+              }}
+            />
+          </EuiFormRow>
+          <EuiFormRow
+            label={
+              <FormattedMessage
+                id="console.settingsPage.refreshingDataLabel"
+                defaultMessage="Refreshing autocomplete suggestions"
+              />
+            }
+            helpText={
+              <FormattedMessage
+                id="console.settingsPage.refreshingDataDescription"
+                defaultMessage="Console refreshes autocomplete suggestions by querying Elasticsearch.
+                  Automatic refreshes may be an issue if you have a large cluster or if you have network limitations."
+              />
+            }
+          >
+            <EuiSwitch
+              checked={polling}
+              data-test-subj="autocompletePolling"
+              id="autocompletePolling"
+              label={
+                <FormattedMessage
+                  defaultMessage="Automatically refresh autocomplete suggestions"
+                  id="console.settingsPage.pollingLabelText"
+                />
+              }
+              onChange={e => setPolling(e.target.checked)}
+            />
+          </EuiFormRow>
+
+          <EuiFlexItem grow={false}>
```

Comment:
Can we remove this wrapping `EuiFlexItem` element? It causes the button to go full-width, which I think over-emphasizes it. This looks better to me:

![image](https://user-images.githubusercontent.com/1238659/60059484-9d08a000-96a1-11e9-8493-e28f02cb9529.png)